src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Blocks/import_panel.html.twig

```diff
@@ -0,0 +1,178 @@
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% trans_default_domain 'Admin.Advparameters.Feature' %}
+
+{% set importForm = form %}
+
+{{ form_start(form, {'attr': { 'data-file-upload-url': file_upload_url, 'class': 'js-import-form' }}) }}
+<div class="card">
+    <h3 class="card-header">
+        <i class="material-icons">import_export</i> {{ 'Import'|trans({}, 'Admin.Actions') }}
+    </h3>
+    <div class="card-block">
+        <div class="alert alert-info" role="alert">
+            <p class="alert-text">
+                {{ 'You can read information on import at:'|trans({}, 'Admin.Advparameters.Help') }}
+                <a href="{{ 'http://doc.prestashop.com/display/PS17/Import'|trans({}, 'Admin.Advparameters.Help') }}" class="_blank">
+                    {{ 'http://doc.prestashop.com/display/PS17/Import'|trans({}, 'Admin.Advparameters.Help') }}
+                </a>
+            </p>
+            <p class="alert-text">
+                {{ 'Read more about the CSV format at:'|trans({}, 'messages') }}
+                <a href="{{ 'https://en.wikipedia.org/wiki/Comma-separated_values'|trans({}, 'Admin.Advparameters.Help') }}" class="_blank">
+                    {{ 'https://en.wikipedia.org/wiki/Comma-separated_values'|trans({}, 'Admin.Advparameters.Help') }}
+                </a>
+            </p>
+        </div>
+
+        <hr>
+
+        <div class="card-text">
+            <div class="form-group">
+                <label class="form-control-label">{{ 'What do you want to import?'|trans }}</label>
+                {{ form_errors(importForm.entity) }}
+                {{ form_widget(importForm.entity, {'attr': {'class': 'js-entity-select'}}) }}
+            </div>
+
+            <div class="alert alert-warning js-entity-alert" role="alert">
+                <ul>
+                    <li>{{ 'Note that the Category import does not support having two categories with the same name.'|trans({}, 'Admin.Advparameters.Notification') }}</li>
+                    <li>{{ 'Note that you can have several products with the same reference.'|trans({}, 'Admin.Advparameters.Notification') }}</li>
+                </ul>
+            </div>
+
+            <hr>
+
+            <div class="form-group hidden-xs-up">
+                {{ form_errors(importForm.csv) }}
+                {{ form_widget(importForm.csv, {'attr': {'class': 'js-import-file-input'}}) }}
+            </div>
+
+            <div class="form-group js-file-upload-form-group">
+                <label class="form-control-label">{{ 'Select a file to import'|trans }}</label>
+                {{ form_errors(importForm.file) }}
+                {{ form_widget(importForm.file, {'attr': {'class': 'js-import-file', 'data-max-file-upload-size': max_file_upload_size }}) }}
+                <span>{{ 'or'|trans({}, 'messages') }}</span>
```

Comment:
Can I have more info about this string ? It can't fall into the messages domain.

Reply:
Agree, the key probably exists in `Admin.Global` domain

Reply:
It does exist in this domain.